### PR TITLE
#167730734 Integrate Coveralls code coverage service

### PR DESCRIPTION
**Pull request by ola357**
#### What does this PR do?
* Integrate Coveralls for code coverage
* Integrate nyc for reporting coverage
#### Description of Task to be completed?
* install nyc and coveralls node packages
* Configure package.json
* run `npm test`
#### How should this be manually tested?
* git clone this branch `ch-coveralls-code-coverage-167730734`
* cd to the project
* Checkout to the branch
* run `npm install` followed by `npm run coverage`
#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
* [#167730734](https://www.pivotaltracker.com/story/show/167730734)
#### Screenshots or gifs (if appropriate)
Screenshot_2019-08-21-00-54-35
#### Questions:
*N/A

### CODE REVIEW

**Review by wahabtobibello — APPROVED** [review]
Good to go!

**Comment by zanio** [comment]
Good Job.

**Review by zanio — APPROVED** [review]
Good job.

**Comment by rovilay** [comment]
@ola357 please ready your `pr description` and correct any grammar blunder.

**Comment by rovilay** [comment]
@ola357 good job. Please add the `coveralls` badge.

**Comment by rovilay** [comment]
@ola357 Please resolve conflicts

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development mocha src/__tests__/* --require @babel/register --require @babel/polyfill",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```
Comment (rovilay) [inline package.json]: @ola357 I get this error when I run coverage 👇🏽
Screenshot 2019-08-20 at 11 06 04 AM
.

Comment (ola357) [reply package.json]: Error has been fixed

Comment (ola357) [reply package.json]: the error has been fixed, PR is ready for merging @rovilay

**Pull request merged**